src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -102,6 +102,8 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         _settingsSource = settings;
         _settingsClone = settings.Copy();
 
+        _updateBackgroundForMica();

```

Comment:
```suggestion
        _UpdateBackgroundForMica();
```
nit: all the other ones start capitalized